monix-eval/shared/src/test/scala/monix/eval/TaskTraverseSuite.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2014-2016 by its authors. Some rights reserved.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.eval
+
+import concurrent.duration._
+import scala.util.{Success, Failure}
+
+object TaskTraverseSuite extends BaseTestSuite {
+  test("Task.traverse should execute in parallel") { implicit s =>
```

Comment:
One small thing - traverse does NOT execute things in parallel. The test code says so as well - if the tasks would execute in parallel, they'd execute in 3 seconds and not 6.

You just need to rename this test - "should" -> "should not"